modules/renter/contractor/consts.go

```diff
@@ -62,7 +67,11 @@ var (
 	maxCollateral    = types.SiacoinPrecision.Mul64(1e3) // 1k SC
 	maxDownloadPrice = maxStoragePrice.Mul64(3 * 4320)
 	maxStoragePrice  = types.SiacoinPrecision.Mul64(30e3).Div(modules.BlockBytesPerMonthTerabyte) // 30k SC / TB / Month
-	maxUploadPrice   = maxStoragePrice.Mul64(4320)
+	maxUploadPrice   = build.Select(build.Var{
+		Standard: maxStoragePrice.Mul64(4320),
+		Dev:      maxStoragePrice.Mul64(4320),
+		Testing:  types.SiacoinPrecision,
```

Comment:
It's hard to tell by reading this whether the testing max price is much smaller or much larger, it should probably match the others but be 4320e6 or something